packages/elements/src/tree/managers/tree-manager.ts

```diff
@@ -40,6 +40,8 @@ export class TreeManager<T extends TreeDataItem> {
   private treeNodeCache = new Map<T, TreeNode<T>>();
 
   /**
+   * A special method to be called when initializes the class.
```

Comment:
it seems this jsdoc comment doesn't get picked up by TypeDoc. I don't think that the comment is important enough to justify the need of scripting enhancement. Let's remove it instead.

Reply:
Updated to fix to display description on doc and rewrite the hole sentences.